backend/cn/locations.mli

```diff
@@ -1,14 +1,25 @@
-(* taking things from ocaml_locations *)
+(* Module Locations -- Utility functions for Cerberus locations
+
+   This module adds a number of useful functions on locations to the
+   ones already provided by Cerberus. *)
 
+(* taking things from ocaml_locations *)
+(* TODO: BCP: What does that mean? *)
 
+(* TODO: BCP: I think this type could be made abstract with only a little
+   work -- its identity seems to be used in just a few places in other
+   modules. *)
 type t = Cerb_location.t
```

Comment:
I'm not sure if that'd be worth it? But I'm open to it.

Reply:
I'll see if it's actually easy.